src/main.cpp

```diff
@@ -12,6 +13,7 @@ int main(int argc, char *argv[])
 
 	qmlRegisterType<PlayerList>("space.developers", 1, 0, "PlayerList");
 	qmlRegisterType<Player>("space.developers", 1, 0, "Player");
+    qmlRegisterType<FileEngine>("space.developers", 1, 0, "FileEngine");
```

Comment:
Похоже на типичного одиночку.
```suggestion
    qmlRegisterSingletonType<FileEngine>("space.developers", 1, 0, "FileEngine", player_api_factory);
```

Reply:
Принято